src/coreComponents/dataRepository/LogLevelsInfo.hpp

```diff
@@ -0,0 +1,193 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file LogLevelsInfo.hpp
+ * This file contains all log level informations and the mecanism to ensure LOG_LEVEL_INFO structure is valid
+ */
+#ifndef GEOS_COMMON_LOGLEVELSINFO_HPP
+#define GEOS_COMMON_LOGLEVELSINFO_HPP
+
+#include "common/DataTypes.hpp"
+#include "common/format/Format.hpp"
+
+namespace geos
+{
+
+namespace logInfo
+{
+
+/**
+ * @name Common LogLevels info structures. They must comply with the `is_log_level_info` trait.
+ */
+///@{
+
+/// @cond DO_NOT_DOCUMENT
+struct LineSearch
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "Information on line search"; }
+};
+
+struct LineSearchFailed
+{
+  static constexpr int getMinLogLevel() { return 1; }
+  static constexpr std::string_view getDescription() { return "On Incorrect solution, Information on failed line search"; }
```

Comment:
this description does not make much sense

Reply:
then i suggest to group everything related to linesearch under the same struct
same for others, try to group as much as possible